PMSCore/Permission.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PMSCore
+{
+
+    /// <summary>
+    /// Represents a user's permission with a level and description.
+    /// </summary>
+    public class Permission
+    {
+        /// <summary>
+        /// Gets the level of the permission.
+        /// A higher value indicates more access or privileges.
+        /// </summary>
+        public int PermissionLevel { get; private set; }
```

Comment:
maybe PermissionLevel could be of a type enum instead of an int what do you think?